fix object ghost follows invisible cursor when scroll with right-click (#17245)
OpenRCT2/OpenRCT2: distribution/changelog.txt

- Fix: [#17104] Changing map size does not invalidate park size.
- Fix: [#17197] Segfault when extracting files from the GOG installer.
- Fix: [#17205] Map generator sometimes crashes when not all standard terrain objects are available.

> - Fix: [#17245] Object ghost and tool tips follows invisible cursor when scroll viewport with right-click 

Reviewer: ```suggestion
- Fix: [#17245] Object ghosts and tooltips follow invisible cursor when moving the viewport by right-click dragging.
```
